corehq/apps/app_manager/suite_xml/case_tile_templates/bha_referrals.xml

```diff
@@ -0,0 +1,178 @@
+
+<detail id="m12_case_short">
+    <title>
+        <text>
+            <locale id="cchq.case"/>
+        </text>
+    </title>
+
+   <field>
+    <style horz-align="left" vert-align="center" font-size="small">
+      <grid grid-height="1" grid-width="12" grid-x="0" grid-y="0"/>
+    </style>
+    <header>
+      <text/>
+    </header>
+    <template form="plain">
+      <text>
+        <xpath function="concat('**Profile Information**')"/>
```

Comment:
I'm not sure where the invisible character is needed exactly and I didn't see it in most other templates, so I didn't add it yet. Does it need to go on every text field (that's not actually an image)?

Reply:
I'm seeing them in other templates. Them being this:" "

Reply:
like here https://github.com/dimagi/commcare-hq/blob/563e37664c3e2d69e7dfdaef1e7e9f22e8e82358/corehq/apps/app_manager/suite_xml/case_tile_templates/icon_text_grid.xml#L147

Reply:
ya it goes on every non image field. (I saw you just made the change but I already had this typed out so i'll just comment it)

Without it, there wouldn't be space between the field name and value. For example, without it, if the "display text" is "NAME:", it'll be

NAME:Evan
instead of 
NAME: Evan